src/components/Tooltip/index.tsx

```diff
@@ -28,21 +28,20 @@ interface TooltipProps {
   side?: TooltipPrimitive.TooltipContentProps['side']
   content?: React.ReactNode
   sideOffset?: number
-  className?: string
 }
 
 const Tooltip = React.forwardRef<
   React.ElementRef<typeof TooltipPrimitive.Content>,
   TooltipProps
->(({ children, side, content, sideOffset, className }, ref) => {
```

Comment:
I've removed the `className` prop so that consumers won't be able to alter the tooltip styles.